src/win/fs-event.c

```diff
@@ -225,11 +225,11 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   }
 
   if (!handle->buffer) {
-    handle->buffer = (char*)_aligned_malloc(uv_directory_watcher_buffer_size,
-                                            sizeof(DWORD));
```

Comment:
Yeah, it's unnecessary. Feel free to send a PR that changes this code to use ordinary malloc().